contracts/MarketOracle.sol

```diff
@@ -30,15 +30,17 @@ contract MedianOracle is Ownable, IOracle {
 
     // Reports indexed by provider address. Report.timestamp > 0 indicates entry
     // existence.
-    mapping (address => Report) public providerReports;
+    mapping (address => Report[2]) public providerReports;
```

Comment:
The comment `Report.timestamp > 0 indicates entry existence` is confusing since `purgeReports` sets the timestamps to 1.

It looks like `providerReports[provider][0].timestamp > 0` indicates provider existence.

Reply:
By entry I meant entry in the map as in provider, changed it to provider to be more clear.

Reply:
One last thing... the first timestamp, the second, or both have to be > 0?